schemas/includes/related-registration.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A related entity registration allows multiple companies to be linked",
+  "type": "object",
+  "properties": {
+    "registration_type": {
+      "description": "The type of the related registration. Use 'predecessor' to link to the prior registration, 'successor' to link to the replacement registration or 'alternate' to link to an alternative registration, e.g. for companies registered more than once in a jurisdiction.",
+      "enum": [
+        "predecessor",
+        "successor",
+        "alternate"
+      ]
+    },
+    "related_entity": {
```

Comment:
Does it need to be an entity rather than a company?

Reply:
Probably not. Will push change